system-int-test/test/cmr/system_int_test/ingest/deleted_granules_index_test.clj

```diff
@@ -117,6 +117,14 @@
         {:revision_date (t/minus- (t/now) (t/days 1))}
         [granule1-prov1 granule2-prov1 granule3-prov1-2 granule4-prov2]))
 
+    (testing "content-type in search header"
+      (are3 [params]
```

Comment:
Usually only use are3 when there are multiple similar test scenarios with different parameters. Since this is just a single test you can just:

```
(testing "Deleted granule search header contains application/json content-type"
  (let [results (search/find-deleted-granules {:revision_date (t/minus- (t/now) (t/days 1))}
    (is (= "application/json; charset=utf-8"
              (get-in results [:headers :content-type]))))
```

Reply:
Sure. Thanks. Done.